tidy3d/components/data/sim_data.py

```diff
@@ -154,8 +154,11 @@ def source_spectrum_fn(freqs):
             """Source amplitude as function of frequency."""
             spectrum = source_time.spectrum(times, freqs, dt, complex_fields)
 
-            # remove user defined phase from normalization so its effect is present in the result
-            return spectrum * np.conj(user_defined_phase)
+            # Remove user defined amplitude and phase from the normalization
+            # such that they would still have an effect on the output fields.
+            # In other words, we are only normalizing out the arbitrary part of the spectrum
+            # that depends on things like freq0, fwidth and offset.
+            return spectrum / source_time.amplitude / user_defined_phase
```

Comment:
why is the user defined phase divided out here instead of multiplied after np.conj()?

Reply:
It's equivalent, I just thought it makes it clear what we're doing (basically removing a user-defined complex number out of the spectrum).

It seems that `user_defined_phase` is actually not used anywhere else, so I could just make that line 

```
return spectrum / source_time.amplitude / np.exp(1j * source_time.phase)
```

Reply:
Note: updated the code as above.